src/components/manager.ts

```diff
@@ -708,8 +708,8 @@ export class Manager {
                 !fs.existsSync(inputFile)) {
                 continue
             }
-            // Drop the current rootFile often listed as INPUT and drop any file that is already in the texFileTree
-            if (texFile === inputFile || inputFile in this.cachedContent) {
+            if (inputFile === texFile && this.filesWatched.has(inputFile)) {
```

Comment:
Why is `&&` used instead of `||` in the condition?

Reply:
Because it is an error. Thanks a lot for catching this one!